src/assets/css/table.css

```diff
@@ -1,119 +1,183 @@
+    }
 
-.table .input-body-container *::selection {
-    background-color: $brand-color-primary;
-    color: #fff;
-}
+    .input-body-container *::selection {
+        background-color: $brand-color-primary;
+        color: #fff;
+    }
 
-.table tbody+tbody {
-    border-top: 0;
-}
+    &.table-included {
+        background-color: transparent;
+        height: 100%;
+        
+        td {
+            height: 22px;
+            width: 50%;
+        }
+    }
 
-.table thead th {
-    border: 0;
-    background-color: $brand-color-secondary;
-    color: #fff;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    vertical-align: middle;
-}
+    &.table-bordered-horizontally td {
+        border-bottom: 1px solid $brand-border-color;
+    }
 
-.table thead .sort-menu .th-caption {
-    text-overflow: ellipsis;
-    overflow: hidden;
-    display: inline-block;
-    width: 100%;
-}
+    &.table-bordered-vertically td:not(:last-child) {
+        border-right: 1px solid $brand-border-color;
+    }
 
-.table .col-xs-12,
-.table .col-sm-12 {
-    position: static;
-}
+    &.table-bordered-vertically .item-caption td {
+        border-right: none;
+    }
 
-.table.table-bordered-horizontally td {
-    border-bottom: 1px solid $brand-border-color;
-}
+    &.table-bordered-vertically td.indented {
+        line-height: 28px;
+        min-width:0;
+        border-right:0;
+        padding-top:0;
+        padding-bottom:0;
+        vertical-align: middle;
+    }
 
-.table.table-bordered-vertically td:not(:last-child) {
-    border-right: 1px solid $brand-border-color;
-}
+    tbody+tbody {
+        border-top: 0;
+    }
 
-.table.table-bordered-vertically .item-caption td {
-    border-right: none;
-}
+    thead {
+        th {
+            border: 0;
+            background-color: $brand-color-secondary;
+            color: #fff;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow: hidden;
+            vertical-align: middle;
+        }
 
-.table-hide-property {
-    display: none;
-}
+        .sort-menu .th-caption {
+            text-overflow: ellipsis;
+            overflow: hidden;
+            display: inline-block;
+            width: 100%;
+        }
+    }
+
+    .col-xs-12, .col-sm-12 {
+        position: static;
+    }
+
+    td, th {
+        padding: .5rem .5rem;
+        height: 28px;
+    }
+
+    th {
+        font-weight: 500;
+        position: relative;
+    }
+
+    td {
+        width: 75px;
+        position: relative;
+        cursor: pointer;
+        border-top: 0;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        padding-top: 0;
+        padding-bottom: 0;
+        vertical-align: middle;
+
+        span {
+            pointer-events: none;
+        }
+    }
+
+    /* Table cols sizing */
+    .td-lg {
+        width: 225px;
+        max-width: 225px;
+        min-width: 225px;
+
+        .input-dropdown-list {
```

Comment:
@PabloSichert you recently removed the rule that was setting this width to 225px, but there was also another one that set it to auto. Does this collide with your changes ?

Reply:
I didn't test for functionality there. Just randomly discovered and removed an obsolete rule that would never apply. It was something like

```css
.x .y {
  width: 123px;
}

.x .y {
  width: auto;
}
```

Reply:
Yeah, it was a dupe. Ok.